### DISCUSSION TITLE
Change background hex

### DISCUSSION THREAD

**Discussion by kaspardavid** [google/model-viewer#4719] [Q&A]
Hi
This may have already been addressed here, i just ask for patience for a newbie.
I wish to edit the scene background color but couldn’t find it on the site and view the results myself.  Is the only possibility to set a div with a specific hex value or is there a simpler way (I guess so) to achieve the same result.
Thank you for advices
Best.

**Comment by elalish** [comment]
The `<model-viewer>` element inherits all the normal HTML element API including `style`. You can just set `background-color: #eeeeee;` in CSS like any other element.

**Comment by kaspardavid** [comment]
Thank You!
